### PR TITLE
Set Up Sat 6.2 REPOS and REPOSET

### LINKED ISSUE #3758: Rolling Back REPOS and REPOSET on Master branch
related to #3757

### PR DESCRIPTION
**Pull request by renzon**
close #3758

Master branch must contain only last sat version repos

### CODE REVIEW

**Comment by renzon** [comment]
Test using repos:

```console
47 tests deselected by '-k ActivationKeyTestCase and test_positive_add_redhat_product' 
================== 1 passed, 47 deselected in 608.32 seconds ===================
```

File: robottelo/constants.py [hunk 1]
```diff
@@ -354,7 +314,8 @@
 FAKE_3_YUM_REPO = u'http://omaciel.fedorapeople.org/fakerepo01'
 FAKE_4_YUM_REPO = u'http://omaciel.fedorapeople.org/fakerepo02'
 FAKE_5_YUM_REPO = u'http://{0}:{1}@rplevka.fedorapeople.org/fakerepo01/'
-FAKE_6_YUM_REPO = u'https://jlsherrill.fedorapeople.org/fake-repos/needed-errata/'  # noqa
+FAKE_6_YUM_REPO = u'https://jlsherrill.fedorapeople.org/fake-repos/needed' \
```
Comment (rochacbruno) [inline robottelo/constants.py]: The line formatting here can be
```python
FAKE_6_YUM_REPO = (
    u'https://jlsherrill.fedorapeople.org/fake-repos/needed-errata/'
)
```

and then we don't need the #noqa

**Comment by renzon** [comment]
Coverage decreased because number of lines decreased. Check the file I changed has 100% coverage

**Comment by sghai** [comment]
ACK.. 

Please note that this change is against master branch and supports latest released Sat 6.2 version specifically for tools repos.

@oshtaier @abalakh @kbidarkar @lpramuk @rplevka : Please 'ACK'  this change once you'r done with review.

**Comment by kbidarkar** [comment]
ACK

**Comment by sghai** [comment]
ACK

**Pull request merged**